---
title: "'GoTo <labelname>' no es válida porque '<labelname>' está dentro de una instrucción 'Try', 'Catch' o 'Finally' que no contiene esta instrucción"
ms.date: 07/20/2015
f1_keywords:
- bc30754
- vbc30754
helpviewer_keywords:
- BC30754
ms.assetid: 2eefc7fb-fdf0-41e9-bf60-c3bc93580e14
ms.openlocfilehash: 54fbd1852d60605290bef7816b05f7b9ffc8c673
ms.sourcegitcommit: f8c270376ed905f6a8896ce0fe25b4f4b38ff498
ms.translationtype: MT
ms.contentlocale: es-ES
ms.lasthandoff: 06/04/2020
ms.locfileid: "84412551"
---
# <a name="goto-labelname-is-not-valid-because-labelname-is-inside-a-try-catch-or-finally-statement-that-does-not-contain-this-statement"></a>'GoTo \<labelname>' no es válida porque '\<labelname>' está dentro de una instrucción 'Try', 'Catch' o 'Finally' que no contiene esta instrucción
No se puede crear una rama en un bloque `Try...Catch...Finally` .  
  
 **Identificador de error:** BC30754  
  
## <a name="to-correct-this-error"></a>Para corregir este error  
  
- Reestructure el código para que la etiqueta preceda al bloque `Try...Catch...Finally` .  
  
## <a name="see-also"></a>Consulte también

- [Instrucción Try...Catch...Finally](../language-reference/statements/try-catch-finally-statement.md)
